packages/tools/src/tools/segmentation/BrushTool.ts

```diff
@@ -235,11 +233,9 @@ class BrushTool extends BaseTool {
       // and should throw an error or maybe simply just allow circle manipulation
       // and not sphere manipulation
       if (this.configuration.activeStrategy.includes('SPHERE')) {
```

Comment:
Can you assign an error message on the activeStrategy object that can be thrown instead?  Eg, something like:
if( this.getStrategy().VOLUMETRIC_ERROR_MESSAGE ) {
throw new Error(this.getStrategy().VOLUMETRIC_ERROR_MESSAGE );

That way you don't need to know anything special about the strategy, just that it doesn't work unless volumetric.  I really hate including the string key here as it makes it very brittle on changes or additions.